resources/views/layouts/Admin-LTE-2-3-4/master.blade.php

```diff
@@ -86,4 +82,4 @@
         ga('send', 'pageview');
     </script>
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Please revert this file. This change is already on master.
Please merge master branch before.